### ISSUE TITLE
libgit2 binaries are not compatible with current stable Debian

### ISSUE THREAD

**Issue by icedream** [icedream/citizenmp-server-updater#2]
Updater version:
```
1.0.3 (63a006b)
```

OS information:
```
PRETTY_NAME="Debian GNU/Linux 7 (wheezy)"
NAME="Debian GNU/Linux"
VERSION_ID="7"
VERSION="7 (wheezy)"
ID=debian
ANSI_COLOR="1;31"
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support/"
BUG_REPORT_URL="http://bugs.debian.org/"
```

Mono version:
```
Mono JIT compiler version 3.10.0 (tarball Wed Nov  5 12:50:04 UTC 2014)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       altstack
        Notifications: epoll
        Architecture:  amd64
        Disabled:      none
        Misc:          softdebug
        LLVM:          supported, not enabled.
        GC:            sgen
```

Log: (Command line was ```MONO_LOG_LEVEL=debug mono citimp_upd.exe test```)
```
Mono: Config attempting to parse: '/etc/mono/config'.
Mono: Config attempting to parse: '/home/icedream/.mono/config'.
Mono: Assembly Loader probing location: '/usr/lib/mono/4.5/mscorlib.dll'.
Mono: Image addref mscorlib[0xd82a10] -> /usr/lib/mono/4.5/mscorlib.dll[0xd81b00]: 2
Mono: AOT module '/usr/lib/mono/4.5/mscorlib.dll.so' not found: /usr/lib/mono/4.5/mscorlib.dll.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: '/usr/lib/mono/4.5/mscorlib.dll'.
Mono: Config attempting to parse: '/usr/lib/mono/4.5/mscorlib.dll.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/mscorlib/mscorlib.config'.
Mono: Assembly mscorlib[0xd82a10] added to domain citimp_upd.exe, ref_count=1
Mono: Assembly Loader probing location: 'citimp_upd.exe'.
Mono: Image addref citimp_upd[0xded3c0] -> /home/icedream/citimp_test/citimp_upd.exe[0xd80820]: 3
Mono: Assembly citimp_upd[0xded3c0] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module '/home/icedream/citimp_test/citimp_upd.exe.so' not found: /home/icedream/citimp_test/citimp_upd.exe.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: 'citimp_upd.exe'.
Mono: Config attempting to parse: '/home/icedream/citimp_test/citimp_upd.exe.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/citimp_upd/citimp_upd.config'.
Mono: Assembly Loader probing location: 'citimp_upd.exe'.
Mono: AOT module '/home/icedream/citimp_test/citimp_upd.exe.so' not found: /home/icedream/citimp_test/citimp_upd.exe.so: cannot open shared object file: No such file or directory

Mono: Assembly Ref addref citimp_upd[0xded3c0] -> mscorlib[0xd82a10]: 2
Mono: Assembly Loader probing location: '/usr/lib/citizenmp_server_updater.dll'.
Mono: Assembly Loader probing location: '/usr/lib/mono/4.5/Facades/citizenmp_server_updater.dll'.
Mono: Assembly Loader probing location: '/usr/lib/citizenmp_server_updater.exe'.
Mono: Assembly Loader probing location: '/usr/lib/mono/4.5/Facades/citizenmp_server_updater.exe'.
Mono: Config attempting to parse: '/home/icedream/citimp_test/citimp_upd.exe.config'.
Mono: Assembly Loader probing location: '/usr/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll'.
Mono: Image addref System[0xe599c0] -> /usr/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll[0xe7f540]: 2
Mono: Assembly System[0xe599c0] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module '/usr/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll.so' not found: /usr/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: '/usr/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll'.
Mono: Config attempting to parse: '/usr/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/System/System.config'.
Mono: Assembly Ref addref citimp_upd[0xded3c0] -> System[0xe599c0]: 2
Mono: Assembly Ref addref System[0xe599c0] -> mscorlib[0xd82a10]: 3
Mono: DllImport attempting to load: '/usr/lib/libMonoPosixHelper.so'.
Mono: DllImport loaded library '/usr/lib/libMonoPosixHelper.so'.
Mono: DllImport searching in: '/usr/lib/libMonoPosixHelper.so' ('/usr/lib/libMonoPosixHelper.so').
Mono: Searching for 'CreateZStream'.
Mono: Probing 'CreateZStream'.
Mono: Found as 'CreateZStream'.
Mono: DllImport searching in: '/usr/lib/libMonoPosixHelper.so' ('/usr/lib/libMonoPosixHelper.so').
Mono: Searching for 'ReadZStream'.
Mono: Probing 'ReadZStream'.
Mono: Found as 'ReadZStream'.
Mono: DllImport searching in: '/usr/lib/libMonoPosixHelper.so' ('/usr/lib/libMonoPosixHelper.so').
Mono: Searching for 'CloseZStream'.
Mono: Probing 'CloseZStream'.
Mono: Found as 'CloseZStream'.
Mono: Image addref citizenmp_server_updater[0xe86cd0] -> data-0x7fa00014d010[0xeb6860]: 1
Mono: Assembly citizenmp_server_updater[0xe86cd0] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module 'data-0x7fa00014d010.so' not found: data-0x7fa00014d010.so: cannot open shared object file: No such file or directory

Mono: Assembly Ref addref citizenmp_server_updater[0xe86cd0] -> mscorlib[0xd82a10]: 4
Mono: Assembly Loader probing location: '/usr/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll'.
Mono: Image addref System.Core[0xedb090] -> /usr/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll[0xecb8e0]: 2
Mono: Assembly System.Core[0xedb090] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module '/usr/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll.so' not found: /usr/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: '/usr/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll'.
Mono: Config attempting to parse: '/usr/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/System.Core/System.Core.config'.
Mono: Assembly Ref addref citizenmp_server_updater[0xe86cd0] -> System.Core[0xedb090]: 2
Mono: Assembly Ref addref System.Core[0xedb090] -> mscorlib[0xd82a10]: 5
Mono: DllImport attempting to load: '__Internal'.
Mono: DllImport loaded library '(null)'.
Mono: DllImport searching in: '__Internal' ('(null)').
Mono: Searching for 'MoveFileEx'.
Mono: Probing 'MoveFileExW'.
Mono: Probing 'MoveFileExW'.
Mono: Probing 'MoveFileEx'.
Mono: Probing 'MoveFileEx'.
Mono: DllImport searching in: '__Internal' ('(null)').
Mono: Searching for 'SetDllDirectory'.
Mono: Probing 'SetDllDirectoryW'.
Mono: Probing 'SetDllDirectoryW'.
Mono: Probing 'SetDllDirectory'.
Mono: Probing 'SetDllDirectory'.
Mono: DllImport attempting to load: 'kernel32'.
Mono: DllImport error loading library './libkernel32': './libkernel32: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library './libkernel32.so': './libkernel32.so: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libkernel32': 'libkernel32: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libkernel32.so': 'libkernel32.so: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'kernel32': 'kernel32: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library './libkernel32': './libkernel32: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library './libkernel32.so': './libkernel32.so: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libkernel32': 'libkernel32: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libkernel32.so': 'libkernel32.so: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libkernel32': 'libkernel32: cannot open shared object file: No such file or directory'.
Mono: DllImport unable to load library 'libkernel32: cannot open shared object file: No such file or directory'.
Mono: Assembly Loader probing location: '/usr/lib/LibGit2Sharp.dll'.
Mono: Assembly Loader probing location: '/usr/lib/mono/4.5/Facades/LibGit2Sharp.dll'.
Mono: Assembly Loader probing location: '/usr/lib/LibGit2Sharp.exe'.
Mono: Assembly Loader probing location: '/usr/lib/mono/4.5/Facades/LibGit2Sharp.exe'.
Mono: Assembly Loader probing location: '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2sharp.dll'.
Mono: Image addref LibGit2Sharp[0xf199e0] -> /tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2sharp.dll[0xf18ce0]: 2
Mono: Assembly LibGit2Sharp[0xf199e0] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2sharp.dll.so' not found: /tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2sharp.dll.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2sharp.dll'.
Mono: Config attempting to parse: '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2sharp.dll.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/LibGit2Sharp/LibGit2Sharp.config'.
Mono: Assembly Ref addref citizenmp_server_updater[0xe86cd0] -> LibGit2Sharp[0xf199e0]: 2
Mono: Assembly Ref addref LibGit2Sharp[0xf199e0] -> mscorlib[0xd82a10]: 6
Mono: Assembly Ref addref citizenmp_server_updater[0xe86cd0] -> System[0xe599c0]: 3
Mono: Assembly Loader probing location: '/usr/lib/mono/gac/Mono.Posix/4.0.0.0__0738eb9f132ed756/Mono.Posix.dll'.
Mono: Image addref Mono.Posix[0xf059f0] -> /usr/lib/mono/gac/Mono.Posix/4.0.0.0__0738eb9f132ed756/Mono.Posix.dll[0xed9b50]: 2
Mono: Assembly Mono.Posix[0xf059f0] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module '/usr/lib/mono/gac/Mono.Posix/4.0.0.0__0738eb9f132ed756/Mono.Posix.dll.so' not found: /usr/lib/mono/gac/Mono.Posix/4.0.0.0__0738eb9f132ed756/Mono.Posix.dll.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: '/usr/lib/mono/gac/Mono.Posix/4.0.0.0__0738eb9f132ed756/Mono.Posix.dll'.
Mono: Config attempting to parse: '/usr/lib/mono/gac/Mono.Posix/4.0.0.0__0738eb9f132ed756/Mono.Posix.dll.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/Mono.Posix/Mono.Posix.config'.
Mono: Assembly Ref addref citizenmp_server_updater[0xe86cd0] -> Mono.Posix[0xf059f0]: 2
Mono: Assembly Ref addref Mono.Posix[0xf059f0] -> mscorlib[0xd82a10]: 7
Mono: Assembly Loader probing location: '/usr/lib/mono/gac/Microsoft.Build.Framework/4.0.0.0__b03f5f7f11d50a3a/Microsoft.Build.Framework.dll'.
Mono: Image addref Microsoft.Build.Framework[0xf2bc20] -> /usr/lib/mono/gac/Microsoft.Build.Framework/4.0.0.0__b03f5f7f11d50a3a/Microsoft.Build.Framework.dll[0xf2ab10]: 2
Mono: Assembly Microsoft.Build.Framework[0xf2bc20] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module '/usr/lib/mono/gac/Microsoft.Build.Framework/4.0.0.0__b03f5f7f11d50a3a/Microsoft.Build.Framework.dll.so' not found: /usr/lib/mono/gac/Microsoft.Build.Framework/4.0.0.0__b03f5f7f11d50a3a/Microsoft.Build.Framework.dll.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: '/usr/lib/mono/gac/Microsoft.Build.Framework/4.0.0.0__b03f5f7f11d50a3a/Microsoft.Build.Framework.dll'.
Mono: Config attempting to parse: '/usr/lib/mono/gac/Microsoft.Build.Framework/4.0.0.0__b03f5f7f11d50a3a/Microsoft.Build.Framework.dll.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/Microsoft.Build.Framework/Microsoft.Build.Framework.config'.
Mono: Assembly Ref addref citizenmp_server_updater[0xe86cd0] -> Microsoft.Build.Framework[0xf2bc20]: 2
Mono: Assembly Ref addref Microsoft.Build.Framework[0xf2bc20] -> mscorlib[0xd82a10]: 8
Mono: Assembly Loader probing location: '/usr/lib/mono/gac/CommandLine/1.9.71.2__de6f01bd326f8c32/CommandLine.dll'.
Mono: Assembly Loader probing location: '/usr/lib/CommandLine.dll'.
Mono: Assembly Loader probing location: '/usr/lib/mono/4.5/Facades/CommandLine.dll'.
Mono: Assembly Loader probing location: '/usr/lib/mono/gac/CommandLine/1.9.71.2__de6f01bd326f8c32/CommandLine.exe'.
Mono: Assembly Loader probing location: '/usr/lib/CommandLine.exe'.
Mono: Assembly Loader probing location: '/usr/lib/mono/4.5/Facades/CommandLine.exe'.
Mono: Assembly Loader probing location: '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/commandline.dll'.
Mono: Image addref CommandLine[0xf2faf0] -> /tmp/costura~7998742135D4EBB63F57486BAEA669DA/commandline.dll[0xf2ecf0]: 2
Mono: Assembly CommandLine[0xf2faf0] added to domain citimp_upd.exe, ref_count=1
Mono: AOT module '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/commandline.dll.so' not found: /tmp/costura~7998742135D4EBB63F57486BAEA669DA/commandline.dll.so: cannot open shared object file: No such file or directory

Mono: Assembly Loader loaded assembly from location: '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/commandline.dll'.
Mono: Config attempting to parse: '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/commandline.dll.config'.
Mono: Config attempting to parse: '/etc/mono/assemblies/CommandLine/CommandLine.config'.
Mono: Assembly Ref addref citizenmp_server_updater[0xe86cd0] -> CommandLine[0xf2faf0]: 2
Mono: Assembly Ref addref CommandLine[0xf2faf0] -> mscorlib[0xd82a10]: 9
Mono: Assembly Ref addref LibGit2Sharp[0xf199e0] -> System.Core[0xedb090]: 3
Mono: Assembly Ref addref LibGit2Sharp[0xf199e0] -> System[0xe599c0]: 4
Mono: DllImport attempting to load: '/usr/lib/libMonoPosixHelper.so'.
Mono: DllImport loaded library '/usr/lib/libMonoPosixHelper.so'.
Mono: DllImport searching in: '/usr/lib/libMonoPosixHelper.so' ('/usr/lib/libMonoPosixHelper.so').
Mono: Searching for 'Mono_Posix_Syscall_stat'.
Mono: Probing 'Mono_Posix_Syscall_stat'.
Mono: Found as 'Mono_Posix_Syscall_stat'.
Mono: Assembly Ref addref CommandLine[0xf2faf0] -> System.Core[0xedb090]: 4
Mono: DllImport attempting to load: 'git2-91fa31f'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f': '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f.so': '/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.15' not found (required by /tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f.so)'.
Mono: DllImport error loading library 'libgit2-91fa31f': 'libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libgit2-91fa31f.so': 'libgit2-91fa31f.so: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'git2-91fa31f': 'git2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f': '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f.so': '/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.15' not found (required by /tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f.so)'.
Mono: DllImport error loading library 'libgit2-91fa31f': 'libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libgit2-91fa31f.so': 'libgit2-91fa31f.so: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libgit2-91fa31f': 'libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport unable to load library 'libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport attempting to load: 'git2-91fa31f'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f': '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f.so': '/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.15' not found (required by /tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f.so)'.
Mono: DllImport error loading library 'libgit2-91fa31f': 'libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'libgit2-91fa31f.so': 'libgit2-91fa31f.so: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library 'git2-91fa31f': 'git2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f': '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/libgit2-91fa31f: cannot open shared object file: No such file or directory'.
Mono: DllImport error loading library '/tmp/costura~7998742135D4EBB63F57486BAEA669DA/l
Unhandled Exception:
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeInitializationException: An exception was thrown by the type initializer for LibGit2Sharp.Core.NativeMethods ---> System.DllNotFoundException: git2-91fa31f
  at (wrapper managed-to-native) LibGit2Sharp.Core.NativeMethods:git_threads_init ()
  at LibGit2Sharp.Core.NativeMethods+LibraryLifetimeObject..ctor () [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Core.NativeMethods..cctor () [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at LibGit2Sharp.Core.Handles.SafeHandleBase..ctor () [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Core.Handles.NullRepositorySafeHandle..ctor () [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Core.Proxy.git_repository_open_ext (System.String path, RepositoryOpenFlags flags, System.String ceilingDirs) [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Repository.IsValid (System.String path) [0x00000] in <filename unknown>:0 
  at CitizenMP.Server.Installer.Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <filename unknown>:0 
  at Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeInitializationException: An exception was thrown by the type initializer for LibGit2Sharp.Core.NativeMethods ---> System.DllNotFoundException: git2-91fa31f
  at (wrapper managed-to-native) LibGit2Sharp.Core.NativeMethods:git_threads_init ()
  at LibGit2Sharp.Core.NativeMethods+LibraryLifetimeObject..ctor () [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Core.NativeMethods..cctor () [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at LibGit2Sharp.Core.Handles.SafeHandleBase..ctor () [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Core.Handles.NullRepositorySafeHandle..ctor () [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Core.Proxy.git_repository_open_ext (System.String path, RepositoryOpenFlags flags, System.String ceilingDirs) [0x00000] in <filename unknown>:0 
  at LibGit2Sharp.Repository.IsValid (System.String path) [0x00000] in <filename unknown>:0 
  at CitizenMP.Server.Installer.Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <filename unknown>:0 
  at Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
```

**Comment by icedream** [comment]
The binaries in the [current develop build 58](http://ci.icedream.kthx.at/view/CitiMP%20Server%20Updater/job/citiserver-updater-develop/55) fix this issue (plus another issue that would have popped up due to a dependency to OpenSSL 0.9.8 on Squeeze) and will be merged into master/release-1.1.

**Comment by icedream** [comment]
Issue fixed in version 1.1.
